lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -346,12 +346,21 @@
           <h2 class="heading -emphasized -inverse"><span><?php print t('Step 4: Prove It'); ?></span></h2>
         </div>
       </div>
+
       <div class="container__row">
           <div class="container__block">
             <h3 class="inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
             <?php if (isset($reportback_copy)): ?>
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
+
+            <h3 class="inline-alt-text-color"><?php print t('Helpful Tips'); ?></h3>
+            <?php if (isset($helpful_tips)): ?>
+              <div class="with-lists">
```

Comment:
But notice the `<ul>` output is _outside_ the `<p>` tags. You shouldn't be putting the `<ul>` inside the `<p>`.

You can probs just use the `<div>` and move the classes on the `<p>` to the `<div>` and just output the `$helpful_tips` inside just the `<div>`. See if that works 😄